website/mkdocs.yml

```
@@ -42,7 +42,7 @@ nav:
       - Custom Stylesheet & JavaScript: topic_userstyle.md
       - Portable Mode: topic_portablemode.md
       - Custom transliteration: topic_transliteration.md
-      - Customize Dictionary Name: custom_dictionary_name.md
+      - Customize Dictionary Name: custom_dictionary.md
```

```suggestion
      - Customize Dictionary: custom_dictionary.md
```

The title is the first part, the second part will decide url,